.github/workflows/echobot-build-BCCB.yml

```diff
@@ -0,0 +1,112 @@
+name: .NET BOT D1 BUILD BCCB
```

Comment:
The Workflow names seem very cryptic to me.

Reply:
The workflows get generated automatically for the user based on a template. This is the name of them template '.NET BOT D1 BUILD {OrgName}'. The OrgName that the user creates will be replaced with that token. D1 is the environment name but the user can update as well. I removed my user workflows.